### DISCUSSION TITLE
Improve search bar

### DISCUSSION THREAD

**Discussion by nardjesse** [JCornat/movie#26] [Ideas]
Would it be possible to change the search bar to add a x icon that empties the field? Like this: 
![image](https://github.com/JCornat/movie/assets/32144769/97fe8d09-7a3e-40a2-b94e-7202208e1d9f)
WDYT?

**Comment by JCornat** [comment]
Yes, great idea !

**Comment by nardjesse** [comment]
Thanks, I'll create a PR for that!
